demos/multi_faces_analysis_pipeline/python/README.md

```diff
@@ -42,13 +42,44 @@ git clone https://github.com/openvinotoolkit/model_server.git
 cd model_server/demos/multi_faces_analysis_pipeline/python
 ```
 
-You can prepare the workspace that contains all the above by just running
+You can prepare the workspace that contains all the above by just running `make` command.
+Since custom node used in this demo is included in OpenVINO Model Server image you can either use the custom node from the image, or build one.
+
+If you just want to quickly run this demo and use already compiled custom node, run: 
 
 ```bash
 make
 ```
 
-### Final directory structure
+#### Directory structure (without custom node)
+
+Once the `make` procedure is finished, you should have `workspace` directory ready with the following content.
+
+```bash
+workspace
+├── age-gender-recognition-retail-0013
+│   └── 1
+│       ├── age-gender-recognition-retail-0013.bin
+│       └── age-gender-recognition-retail-0013.xml
+├── config.json
+├── emotion-recognition-retail-0003
+│   └── 1
+│       ├── emotions-recognition-retail-0003.bin
+│       └── emotions-recognition-retail-0003.xml
+└── face-detection-retail-0004
+    └── 1
+        ├── face-detection-retail-0004.bin
+        └── face-detection-retail-0004.xml
+
+```
+
+If you modified the custom node or for some other reason, you want to have it compiled and then attached to the container, run:
```

Comment:
This won't replace the custom node already present in OVMS container so it will require adjusting config.json as well?

Reply:
Config.json is automatically edited in Makefile via sed